lib/screens/world_clock_widget.dart

```diff
@@ -37,8 +37,8 @@ class _WorldClockWidgetState extends State<WorldClockWidget> {
             },
             child: GestureDetector(
               child: Container(
-                width: 120,
-                height: 90,
+                width: 180,
+                height: 150,
```

Comment:
please use relative width/height `MediaQuery.of(context).size.width/2; ...height/5`

!!! sometimes padding&margin are enough, because 
**_[container fits to child](https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstley)_**